wallet/src/wallet.rs

```diff
@@ -691,12 +662,19 @@ fn process_show_stake_account(
     let stake_account = rpc_client.get_account(staking_account_pubkey)?;
     match stake_account.state() {
         Ok(StakeState::Stake(stake)) => {
-            println!("account lamports: {}", stake_account.lamports);
-            println!("voter pubkey: {}", stake.voter_pubkey);
+            println!("total stake: {}", stake_account.lamports);
             println!("credits observed: {}", stake.credits_observed);
-            println!("stake: {}", stake.stake);
+            println!("delegated stake: {}", stake.stake);
```

Comment:
@rob-solana can you please take a quick peek at this logic where I print out the activation state of the stake, did I read the code right?

Reply:
yep.

Reply:
at some point, we could drop the stake member and use lamports instead